src/current/_includes/releases/v25.2/v25.2.0-alpha.2.md

```diff
@@ -0,0 +1,96 @@
+## v25.2.0-alpha.2
+
+Release Date: March 31, 2025
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v25-2-0-alpha-2-sql-language-changes">SQL language changes</h3>
+
+- `num_nulls()` and `num_nonnulls()` no longer require that all arguments have the same type.
+ [#141193][#141193]
+- `concat()` no longer requires that all arguments have the same type.
+ [#141193][#141193]
+- `pg_column_size()` no longer requires that all arguments have the same type.
+ [#141193][#141193]
+- Users can now begin logical data replication (LDR) on an existing table if the user has a table-level `REPLICATIONDEST` privilege. Furthermore, users can now begin LDR onto an automatically created table if the user has the parent database level `CREATE` privilege. Finally, during bidirectional LDR, the user in the original source URI, who will begin the reverse LDR stream, to authorize via this table-level `REPLICATIONDEST` privilege.
```

Comment:
@katmayb Could you please have a look at this release note? Specifically the last sentence, which I can't quite parse.

Reply:
```suggestion
- Users can now begin logical data replication (LDR) on an existing table if the user has a table-level `REPLICATIONDEST` privilege. Furthermore, users can now begin LDR onto an automatically created table if the user has the parent database level `CREATE` privilege. Finally, during bidirectional LDR, the user in the original source URI, who will begin the reverse LDR stream, will authorize via this table-level `REPLICATIONDEST` privilege.
```